### PR TITLE
Sort the application suggestion names in application filter form

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
Sort the application suggestion names in application filter form
```

### CODE REVIEW

File: web/src/utils/unique-array.ts [hunk 1]
```diff
@@ -1 +1,2 @@
-export const uniqueArray = <T>(arr: Array<T>): Array<T> => [...new Set(arr)];
+export const uniqueArray = <T>(arr: Array<T>): Array<T> =>
+  [...new Set(arr)].sort();
```
Comment (nghialv) [inline web/src/utils/unique-array.ts:2]: It seems that you changed at the shared place.
Will this affect all places where this function is being used? Is that ok?

Comment (khanhtc1202) [reply web/src/utils/unique-array.ts:2]: yes, I confirmed that this one only is used in related places we want. And besides, having a sorted set is a good point as well, so I think it's okay to make change like this.

**Review by knanao — APPROVED** [review]
Nice!

File: web/src/utils/unique-array.ts (hunk 1 above)
Comment (khanhtc1202) [reply web/src/utils/unique-array.ts]: sure, I rename it as sortedSet 🙆‍♀️

**Comment by khanhtc1202** [comment]
@knanao sorry but could you re-approval this PR 🙏

**Review by knanao — APPROVED** [review]
Sure, Here you go!

**Pull request merged**